drivers/SmartThings/zigbee-humidity-sensor/profiles/humidity-temp-battery-aqara.yml

```diff
@@ -0,0 +1,24 @@
+name: humidity-temp-battery-aqara
+components:
+  - id: main
+    capabilities:
+      - id: temperatureMeasurement
+        version: 1
+      - id: relativeHumidityMeasurement
+        version: 1
+      - id: battery
+        version: 1
+      - id: firmwareUpdate
+        version: 1
+      - id: refresh
+        version: 1
+    categories:
+      - name: MultiFunctionalSensor
+preferences:
+  - preferenceId: tempOffset
+    explicit: true
+  - preferenceId: humidityOffset
+    explicit: true
+metadata:
+  mnmn: SmartThingsCommunity
+  vid: c46d0006-c415-3de5-b493-3114b8ebc0fc
```

Comment:
can I ask what you've done here that required a custom metadata? if it was just the icon, you could have changed the `categories` value to `Thermostat`

Reply:
In order to display the temperature and humidity together on the Dashboard UI, a new presentation was created and metadata was set in the profile. [Seong-Yeol-Park](https://github.com/Seong-Yeol-Park) will tell you the details.

Reply:
Hi @greens , 
And the major changes are
1. Show temperature and humidity together at dashboard
2. Change icon: we are trying to add icon url (CAPI-4568). if it doesn't work we can try to change category with `Thermostat`

Reply:
@sbkimaqara,
Could you change the categories value to Thermostat ?
( My guide using iconUrl is not working. I'm sorry )

Reply:
Hi @greens, 

Aqara still need custom metadata for "Show temperature and humidity together at dashboard"
as Kwang-Hui said.

Ref. https://smartthings.atlassian.net/wiki/spaces/DICE/pages/2211950016/Composite+states+for+multiple+capabilities

![image](https://user-images.githubusercontent.com/100197321/192193027-2ef126e9-f643-46a9-8d80-f623fcadf37b.png)

Reply:
> Hi @sbkimaqara, Could you change the categories value to Thermostat ? ( My guide using iconUrl is not working. I'm sorry )

Updated. :)